teuthology/orchestra/remote.py

```diff
@@ -26,7 +26,275 @@
 log = logging.getLogger(__name__)
 
 
-class Remote(object):
+class LocalOps(object):
+    """
+    This class serves a shared interface for teuthology's orchestra and
+    vstart_runner. It contains methods that operate on the given remote
```

Comment:
I don't think vstart_runner should ever mentioned in teuthology code base, because it is a consumer software... we can build the interface universal but teuthology as a framework should not aware about whoever who use it.

Reply:
But in general why not use pure ssh for vstart_runner, maybe it is metter of configuring to localhost for simplification of interface?

Reply:
@kshtsk teuthology is used as a library in the Ceph code base but which parts of it are used is not always clear. I think we should at least emphasize this class is inherited from externally.

Reply:
@batrick 
>  think we should at least emphasize this class is inherited from externally.

Do you mean, "emphasize that this class is inherited externally?" I am confused and don't understand this last part.

From a general perspective, I find myself agreeing with @kshtsk here...

Reply:
> @batrick
> 
> > think we should at least emphasize this class is inherited from externally.
> 
> Do you mean, "emphasize that this class is inherited externally?" I am confused and don't understand this last part.

It means this class is used as an interface in code outside of the teuthology.git code base.

> From a general perspective, I find myself agreeing with @kshtsk here...

It's not always clear in teuthology's code which methods/classes are interacted with externally by ceph qa/ tree. A curious code reader might assume pulling these methods out of Remote was unnecessary. A comment should explain why it's done.

Reply:
Okay. Please let me know if the current comment is enough.